src/decrypt-api/decrypt-api.csproj

```diff
@@ -19,7 +19,6 @@
     <PackageReference Include="App.Metrics.Formatters.Prometheus" Version="3.2.0" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.0.0" />
     <PackageReference Include="KubernetesClient" Version="1.6.1" />
-    <PackageReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Why?

Reply:
You can see in my commit message, the build writes a warning regards that reference is not required on .NETCore 3.